test/verify/check-host-switching

```diff
@@ -258,6 +261,9 @@ class TestHostSwitching(MachineCase, HostSwitcherHelpers):
         b.click("#nav-hosts .nav-item a[href='/@10.111.113.3']")
         b.wait_js_cond('window.location.pathname.indexOf("/@10.111.113.3") === 0')
 
+        # remote hosts should not have the dashboard
+        b.wait_not_in_text("#nav-system", "Dashboard")
```

Comment:
This will most probably fail on -distropkg.

Reply:
Yes, fixed.